docs/integration/circle.md

```diff
@@ -1,3 +1,51 @@
 # Integrating with CircleCI
 
-This is a stub article. Help writing it by clicking "Edit this page on GitHub"
+[CricleCI](https://circleci.com/) is a continous integration platform that allows you test your code and automate workflows in your project. You can use StepCI in CircleCI using the official [Node Orb](https://circleci.com/developer/orbs/orb/circleci/node)
+
+To integrate StepCI into your CircleCI build:
```

Comment:
Write Step CI separately

Reply:
Okay let me change that

Reply:
It runs well in the nodejs orb on CircleCI. I've also tested with the bun orb (with a different config) and that works too.